Samples/outlook-check-item-categories/README.md

```diff
@@ -0,0 +1,222 @@
+
+Smart Alerts and its related events, `OnMessageSend` and `OnAppointmentSend`, are currently available in preview. For documentation related to this sample, see [Use Smart Alerts and the onMessageSend event in your Outlook add-in (preview)](https://docs.microsoft.com/office/dev/add-ins/outlook/smart-alerts-onmessagesend-walkthrough).
+
+## Features
+
+![Screenshot of the sample displaying Outlook Smart Alerts when required categories are missing from the message or appointment, and the task pane used to apply the categories to the item.](./assets/outlook-check-item-categories-features.png)
+
+1. Use event-based activation triggered by the `OnNewMessageCompose` and `OnNewAppointmentOrganizer` events to create color categories based on specific keywords.
+
+1. Use event-based activation triggered by the `OnMessageSend` and `OnAppointmentSend` events to verify that a required category is applied to a new message or appointment for each detected keyword in the item's subject or body.
+
+1. Use Outlook Smart Alerts to block a message or appointment from being sent if it's missing some or all of the required categories.
+
+1. Apply the required categories to the message or appointment using the add-in's task pane.
+
+## Applies to
+
+- Outlook on Windows (minimum build - 16.0.14511.10000)
+
+## Prerequisites
+
+- Microsoft 365
+
+> Note: If you don't have a Microsoft 365 subscription, you can get a [free developer sandbox](https://developer.microsoft.com/microsoft-365/dev-program#Subscription) that provides a renewable 90-day Microsoft 365 E5 subscription for development purposes.
+
+- Outlook on Windows with a minimum build of 16.0.14511.10000. For guidance with client configuration, see [How to preview](https://docs.microsoft.com/office/dev/add-ins/outlook/autolaunch#how-to-preview).
+
+## Solution
+
+| Solution | Author(s) |
+| -------- | --------- |
+| Use Outlook Smart Alerts to verify that required color categories are applied to a new message or appointment before it's sent | Microsoft |
+
+## Version history
+
+| Version | Date | Comments |
+| ------- | ----- | -------- |
+| 1.0 | 05-05-2022 | Initial release |
+
+## Run the sample
+
+Run this sample in Outlook on Windows. The add-in web files are served from this repository on GitHub.
+
+1. Download the **manifest.xml** file from this sample to a folder on your computer.
+
+1. Sideload the add-in manifest in Outlook on Windows by following the manual instructions in [Sideload Outlook add-ins for testing](https://docs.microsoft.com/office/dev/add-ins/outlook/sideload-outlook-add-ins-for-testing).
+
+### Try it out
+
+Once the add-in is loaded, use the following steps to try out its functionality.
+
+1. Create a new message or appointment.
+
+    > Office Add-ins Sample color categories are created. To verify this from the new message or appointment window, select **File** \> **Info** \> **Properties** \> **Categories**.
```

Comment:
```suggestion
    Office Add-ins Sample color categories are created. To verify this from the new message or appointment window, select **File** \> **Info** \> **Properties** \> **Categories**.
```
This right carat character doesn't render correctly in the list. Suggest removing it and manually adding 4 spaces to the second line of the procedure for readability.

Reply:
Thanks, @lindalu-MSFT. Just making sure I'm seeing the same thing you're getting. This is how it appears on my end. Do you still suggest removing the right carat character?

![image](https://user-images.githubusercontent.com/15154970/167505819-61fa2f38-555b-45e1-96a5-1865f34d456e.png)